MAINTAINING.md

```diff
@@ -0,0 +1,23 @@
+# Maintaining UMA Repositories
+
+As our repos grow in activity, it takes more and more effort to maintain them and ensure code and issues are being
+actively triaged for consideration and review. This document outlines the responsibilities of the on-duty maintainer.
+
+## Responsibilities
+
+- Every business day, the on-duty maintainer is expected to make a pass over _all_ open PRs and new issues in _all_
+  UMA repositories.
```

Comment:
should this not just apply to the protocol repo? the rules & process on the UMIP/front end repo will properly need to be different, don't you think?

Reply:
It's a good question. Maybe it shouldn't apply to UMIPs repo, but my feeling is that it _should_ apply to EMP-tools, EMP-launch, docs, etc? Maybe we just mention the exceptions here? Or do you think there's a better way to organize it?

Reply:
We have a separate process managing the UMIP repo so it makes sense to exclude it. I think all of the other repos are applicable. E.g., if it's a doc PR then tag a non-eng to work on it

Reply:
ye, I think it best to scope this to just protocol for now.